src/main/webapp/dev/js/common/feedbackResponseComments.es6

```diff
@@ -497,8 +497,102 @@ function registerResponseCommentCheckboxEvent() {
     });
 }
 
+const deleteCommentHandlerForFeedbackPage = (e) => {
+    const submitButton = $(e.currentTarget);
```

Comment:
I haven't made changes in these file as methods `deleteCommentHandlerForFeedbackPage` and `registerResponseCommentCheckboxEventForFeedbackPage` have been copied with slight changes from methods `deleteCommentHandler` and `registerResponseCommentCheckboxEvent` in the same file.